tools/value_util.go

```diff
@@ -0,0 +1,31 @@
+package tools
+
+import "github.com/arr-ai/arrai/rel"
+
+func ValueAsString(v rel.Value) (string, bool) {
```

Comment:
Add comments to describe these functions. It's relatively clear what they do, but it's not clear why - why would I call `ValueAsString` and not just `value.String()`?

Reply:
As it has to deal with type `rel.GenericSet`. In arr.ai, all empty sets are the same and `""`, `{}` and `[]` will be parsed to `rel.GenericSet`.